src/form-builder/CheckboxFieldGroup.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  useField,
+  FieldHookConfig,
+  FieldArray,
+  useFormikContext,
+} from 'formik';
+
+import {
+  chainArrayValidations,
+  required,
+  requiredLength,
+} from '../utils/validation';
+import CheckboxField from './CheckboxField';
+import { CheckboxGroupProps, CheckboxProps } from './types';
+
+// import CheckboxGroup from 'local-web-components/CheckboxGroup';
+import CheckboxGroup from '@department-of-veterans-affairs/component-library/CheckboxGroup';
+
+const storeInheritedValues = (fieldValue: string[] | undefined) => {
```

Comment:
Could you add a jsdoc comment for `storeInheritedValues` summarizing what it does? Thanks!